cryptofeed/backends/postgres.py

```diff
@@ -8,6 +8,7 @@
 from datetime import datetime as dt
 from typing import Tuple
 
+import logging
```

Comment:
everything else is using the log.py file functionality, why did you opt to use the logging package here?